content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,206 @@
+respective fields and :guilabel:`Save` the settings.
+
+Configure outgoing email server
+-------------------------------
+
+After that, on the :guilabel:`General Settings` page, under :guilabel:`Custom Email Servers`, click
+:guilabel:`Outgoing Email Servers` to configure the external Gmail account.
+
+.. image:: google_oauth/outgoing-servers.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Then, :guilabel:`Create` a new email server and select the option for :guilabel:`Gmail`. Next, fill
+in the :guilabel:`Description` (can be anything) and the email :guilabel:`Username` and click on
+:guilabel:`Connect your Gmail account`.
+
+A new window from :guilabel:`Google` appears to complete the authorization process. Select the
+appropriate email address that is being configured in Odoo.
+
+If the email address is a personal account, an extra step pops up, click :guilabel:`Continue` to
+allow the verification and connect the Gmail account to Odoo.
+
+Then, allow Odoo to access the Google account by clicking on :guilabel:`Continue` or
+:guilabel:`Allow`. After that, the page navigates back to the newly configured outgoing email
+server in Odoo. The configuration automatically loads the token in Odoo, and a tag stating
+:guilabel:`Gmail Token Valid` appears in green.
+
+.. image:: google_oauth/green-token.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Finally, save the settings and :guilabel:`Test the Connection`. The Odoo database can now send
+safe, secure emails through Google using OAuth authentication.
+
+Google OAuth FAQ
+================
+
+Production VS Testing Publishing Status
+---------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of Testing) will
+display the following warning message:
+
+.. image:: google_oauth/published-status.png
+   :class: img-fluid
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
+
+To correct this warning, navigate to the
+`Google API Platform <https://console.cloud.google.com/apis/credentials/consent>`__. If the
+:guilabel:`Publishing status` is :guilabel:`In Production`, click :guilabel:`Back to Testing`
+to correct the issue.
+
+No Test Users Added
+-------------------
+
+If no test users are added to the OAuth consent screen, then a 403 access denied error will
+populate.
+
+.. image:: google_oauth/403-error.png
+   :class: img-fluid
+   :align: center
+   :alt: 403 Access Denied Error.
+
+To correct this error, return to the :guilabel:`OAuth consent screen` under
+:guilabel:`APIs & Services` and add test user(s) to the app. Add the email that you want to
+configure in Odoo.
+
+Gmail Module not updated
+------------------------
+
+If the :guilabel:`Gmail Module` in Odoo has not be updated to the latest version, then a
+:guilabel:`Forbidden` error message populates.
+
+.. image:: google_oauth/forbidden-error.png
```

Comment:
This screenshot is too wide at 988px. Can you retake this screenshot @tiku-odoo ? And remember to compress the PNG image file after you take the screenshot!